### ISSUE TITLE
response is not reading correctly

### ISSUE THREAD

**Issue by emirhannaneli** [shred/acme4j#156]
The locale parameter is required when the toUpperCase method is used in the parsing method of the **org.shredzone.acme4j.Status** class. Otherwise it becomes UNKNOWN

### Currently
```
public static Status parse(String str) {
        String check = str.toUpperCase();
        return (Status)Arrays.stream(values()).filter((s) -> {
            return s.name().equals(check);
        }).findFirst().orElse(UNKNOWN);
 }
```
    
 ### Suggested
```
public static Status parse(String str) {
        String check = str.toUpperCase(Locale.ENGLISH);
        return (Status)Arrays.stream(values()).filter((s) -> {
            return s.name().equals(check);
        }).findFirst().orElse(UNKNOWN);
}
```

**Comment by shred** [comment]
Good catch! Thank you, I will fix that for the next version.

**Comment by emirhannaneli** [comment]
Thanks

**Comment by emirhannaneli** [comment]
> Good catch! Thank you, I will fix that for the next version.

Thanks ^^

**Comment by shred** [comment]
Fixed in acme4j 3.2.1. It's available in the release section and soon on Maven Central. Thanks again!
